schemas/questionnaire_v1.json

```diff
@@ -255,18 +239,27 @@
                 }
               },
               "additionalProperties": false,
-              "required": [
-                "id",
-                "title",
-                "blocks"
-              ]
+              "required": ["id", "title", "blocks"]
             }
+          },
+          "is_enabled": {
```

Comment:
The hub already uses the key "enabled" for the same behaviour. We should use the same key for both to ensure consistency.